crates/biome_graphql_parser/src/parser/parse_error.rs

```diff
@@ -60,3 +60,31 @@ pub(crate) fn expected_root_operation_type_definition(
 pub(crate) fn expected_operation_type(p: &GraphqlParser, range: TextRange) -> ParseDiagnostic {
     expected_any(&["query", "mutation", "subscription"], range, p)
 }
+
+pub(crate) fn expected_directive_location(p: &GraphqlParser, range: TextRange) -> ParseDiagnostic {
+    expected_any(
+        &[
+            "QUERY",
+            "MUTATION",
+            "SUBSCRIPTION",
+            "FIELD",
+            "FRAGMENT_DEFINITION",
+            "FRAGMENT_SPREAD",
+            "INLINE_FRAGMENT",
+            "VARIABLE_DEFINITION",
+            "SCHEMA",
+            "SCALAR",
+            "OBJECT",
+            "FIELD_DEFINITION",
+            "ARGUMENT_DEFINITION",
+            "INTERFACE",
+            "UNION",
+            "ENUM",
+            "ENUM_VALUE",
+            "INPUT_OBJECT",
+            "INPUT_FIELD_DEFINITION",
+        ],
+        range,
+        p,
+    )
```

Comment:
If you need to show a list of alternatives, use this API instead:

https://github.com/biomejs/biome/blob/main/crates%2Fbiome_grit_parser%2Fsrc%2Fparser%2Fparse_error.rs#L66-L69

You get a better looking diagnostic